src/main/java/bio/terra/service/snapshotbuilder/query/filtervariable/FunctionFilterVariable.java

```diff
@@ -43,22 +45,39 @@ public String renderSQL(CloudPlatformWrapper platform) {
   }
 
   public enum FunctionTemplate implements SqlExpression {
-    TEXT_EXACT_MATCH("CONTAINS_SUBSTR(<fieldVariable>, <value>)"),
+    TEXT_EXACT_MATCH(
+        "CONTAINS_SUBSTR(<fieldVariable>, <value>)", "CHARINDEX(<value>, <fieldVariable>) > 0"),
     // BigQuery fuzzy match pattern is "bqutil.fn.levenshtein(UPPER(<fieldVariable>),
     // UPPER(<value>))<5"
     TEXT_FUZZY_MATCH("dbo.Levenshtein(UPPER(<fieldVariable>), UPPER(<value>), 5)"),
     IN("<fieldVariable> IN (<value>)"),
     NOT_IN("<fieldVariable> NOT IN (<value>)");
 
-    private final String template;
+    private final String gcpTemplate;
+    private final String azureTemplate;
 
     FunctionTemplate(String template) {
-      this.template = template;
+      this(template, template);
```

Comment:
is `this(template, template)` a new version of `this.template = template`?

Reply:
this(template, template) references the new constructor below on line 63. So this is saying if only one string template is given, there is no distinction needed for different cloud platforms so just use it for both.

Reply:
Where is the new constructor used in this code? Just in tests so far?

Reply:
It is used on line 48 and 49 of this file to create TEXT_EXACT_MATCH